company.html

```diff
@@ -0,0 +1,77 @@
+<div class="companyBox"  ng-controller="companyCtrl" >
+
+    <div class="searchBox">
+        <span>Wyszukaj firmę: </span>
+        <input class="inputSearch" ng-model="search.nazwa" placeholder="Nawa firmy">
+        <input class="inputSearch" ng-model="search.adres" placeholder="Adres firmy">
+        <input class="inputSearch" ng-model="search.nip" placeholder="Nip firmy">
+        <input class="inputSearch" ng-model="search.nr_konta" placeholder="Numer konta">
+        <span class="addCompany"><button class="buttonAddCompany" ng-click="addDataCompany()">Dodaj firmę</button></span>
+    </div>
+
+    <table>
+        <thead>
+            <th class="thSort">
+                <button  ng-click="sortBy('nazwa')">Nazwa firmy <span class="sortorder" ng-show="propertyName === 'nazwa'" ng-class="{reverse: reverse}"></span></button>
```

Comment:
to akurat pierdola, ale pamietaj ze mozesz pisac wszystkie nazwy programistyczne tylko w jednym jezyku - angielskim albo polskim.  Jesli piszesz:
 propertyName === 'nazwa'
to brzydko to wyglada. 99% sie pisze tylko po angielsku, wiec uzyj tez wszedzie angielskich nazw do sortowania.